basics-assignments-4-start/src/app/my-inner/my-inner.component.ts

```diff
@@ -0,0 +1,36 @@
+// my-inner.component.ts
+
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-my-inner',
+  templateUrl: './my-inner.component.html',
+  styleUrls: ['./my-inner.component.css'],
+})
+export class MyInnerComponent {
+  innerTotal = 5;
```

Comment:
I don't understand what you are trying to do. just get the input. no need to have to variables.